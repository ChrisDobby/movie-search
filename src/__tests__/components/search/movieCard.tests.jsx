import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';
import MovieCard from '../../../components/search/movieCard';

describe('movieCard', () => {
    const movie = {
        id: 1,
        title: 'A Movie',
        posterImage: '1.png',
    };

    it('should render the movie correctly', () => {
        const card = renderer.create((
            <MemoryRouter>
                <MovieCard movie={movie} />
            </MemoryRouter>));

        expect(card.toJSON()).toMatchSnapshot();
    });
});
