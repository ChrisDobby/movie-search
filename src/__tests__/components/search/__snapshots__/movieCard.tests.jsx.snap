// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`movieCard should render the movie correctly 1`] = `
<div
  className="col-lg-3 col-md-4 col-sm-6 col-xs-12"
>
  <div
    className="card"
  >
    <a
      href="/movie/1"
      onClick={[Function]}
    >
      <img
        alt="A Movie"
        className="card-img-top"
        src="1.png"
      />
    </a>
  </div>
</div>
`;
